src/pages/Facility/services/pharmacy/MedicationReturnShow.tsx

```
@@ -0,0 +1,774 @@
+          </div>
+        </div>
+
+        {/* Medication Return Details */}
+        <Card>
+          <CardContent className="space-y-1 p-4">
+            <div className="grid grid-cols-2 md:grid-cols-3 lg:grid-cols-5 gap-3">
+              {deliveryOrder.patient && (
+                <div>
+                  <label className="text-sm font-medium text-gray-700">
+                    {t("patient")}
+                  </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)